src/go/rpk/pkg/cli/cmd/wasm/remove.go

```diff
@@ -74,17 +74,28 @@ func remove(
 		emptyMessage,
 		strconv.FormatUint(id, 10),
 		kafka.CoprocessorTopic,
-		[]sarama.RecordHeader{header},
+		headers,
 	)
 }
 
-func createHeader(action string) sarama.RecordHeader {
-	// create key
-	key := []byte("action")
-	// create value
-	value := []byte(action)
-	return sarama.RecordHeader{
-		Key:	key,
-		Value:	value,
+func createHeader(action string, filename string) []sarama.RecordHeader {
+	var headersResult []sarama.RecordHeader
+	// create simple struct for key string and value string
+	value := []struct {
+		key	string
+		value	string
+	}{
+		{key: "action", value: action},
+		{key: "file_name", value: filename},
 	}
+	// create RecordHeader and append to headersResult
+	for _, v := range value {
+		keyH := []byte(v.key)
+		valueH := []byte(v.value)
+		headersResult = append(headersResult, struct {
+			Key	[]byte
+			Value	[]byte
+		}{Key: keyH, Value: valueH})
+	}
```

Comment:
I don't really get what's going on here 😬 

Why not initialize a single list with the `[]byte` values instead of declaring it with strings and then looping over them and converting them?

Reply:
because no all header has a same key and value